src/System/Wlog/Color.hs

```diff
@@ -11,29 +11,35 @@
 
 module System.Wlog.Color
        ( colorizer
+       , colorizerT
        ) where
 
-import           System.Console.ANSI (Color (Blue, Green, Magenta, Red, Yellow),
-                                      ColorIntensity (Vivid), ConsoleLayer (Foreground),
-                                      SGR (Reset, SetColor), setSGRCode)
+import qualified Data.Text            as T
+import           System.Console.ANSI  (Color (Blue, Green, Magenta, Red, Yellow),
+                                       ColorIntensity (Vivid), ConsoleLayer (Foreground),
+                                       SGR (Reset, SetColor), setSGRCode)
+import           Universum
 
-import           System.Log.Logger   (Priority (DEBUG, ERROR, INFO, NOTICE, WARNING))
+import           System.Wlog.Severity (Severity (..))
 
 -- | Defines pre- and post-printed characters for printing colorized text.
-table :: Priority -> (String, String)
-table priority = case priority of
-    ERROR   -> (setColor Red     , reset)
-    DEBUG   -> (setColor Green   , reset)
-    NOTICE  -> (setColor Magenta , reset)
-    WARNING -> (setColor Yellow  , reset)
-    INFO    -> (setColor Blue    , reset)
-    _       -> ("", "")
+table :: Severity -> (String, String)
+table severity = case severity of
+    Error   -> (setColor Red     , reset)
+    Debug   -> (setColor Green   , reset)
+    Notice  -> (setColor Magenta , reset)
+    Warning -> (setColor Yellow  , reset)
+    Info    -> (setColor Blue    , reset)
   where
     setColor color = setSGRCode [SetColor Foreground Vivid color]
     reset = setSGRCode [Reset]
 
--- | Colorizes text.
-colorizer :: Priority -> String -> String
-colorizer pr s = before ++ s ++ after
-  where
-    (before, after) = table pr
+-- | Colorizes string text.
+colorizer :: Severity -> String -> String
```

Comment:
I still don't think that importing `serokell-util` is good, because it (once again):
1. Has a lot of dependencies
2. Lot of them are not in any LTS
3. It's not in LTS itself.

I find copiyng small fragments of libraries a good decision in some cases, this one counts.